Library/Aliases/aws-elasticbeanstalk

```diff
@@ -0,0 +1 @@
+../Formula/awsebcli.rb
```

Comment:
Please do this other way around. Keep the original file name. And create symlink `Alias/awsebcli`